packages/runtime/src/runtime/hooks/useNavigateModal.tsx

```diff
@@ -56,7 +56,7 @@ export const useNavigateModalScreen: EnsembleActionHook<
       const inputs =
         !isString(action) && action.inputs ? cloneDeep(action.inputs) : {};
       if (screenContext) {
-        const expressionMap: string[][] = [];
+        const expressionMap: [string, string, object][] = [];
```

Comment:
Why is this needed?

Reply:
@evshi previously is was the array of string only, and I need to push the object in that array whenever it find the expression

Reply:
Why do you need the object...?

Reply:
You mean, the types is wrong or I can't use object whenever it found the expression ?